### PR TITLE
Task: Initial

### PR DESCRIPTION
**Pull request by omerk2511**
Created a very basic TCP server implementation for the rest of the project to build upon.

### CODE REVIEW

File: source/handlers/client.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import socket
+import json
+from threading import Thread, Lock
+
+from ..requests.utils import validate_request
+
+BUFFER_SIZE = 4096
```
Comment (Nisitay) [inline source/handlers/client.py:7]: Too big

File: source/handlers/client.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
```
Comment (Nisitay) [inline source/handlers/client.py:7]: too big

**Review by Nisitay — APPROVED** [review]

File: source/handlers/client.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import socket
+import json
+from threading import Thread, Lock
+
+from ..requests.utils import validate_request
+
+BUFFER_SIZE = 1024
```
Comment (yaimgod) [inline source/handlers/client.py:7]: too small

Comment (omerk2511) [reply source/handlers/client.py:7]: lol

Comment (Nisitay) [reply source/handlers/client.py:7]: nice

**Review by yaimgod — APPROVED** [review]

**Pull request merged**